utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -0,0 +1,209 @@
+ANSI_COLOR_BRIGHT_RED=91
+ANSI_COLOR_BRIGHT_GREEN=92
+ANSI_COLOR_BRIGHT_YELLOW=93
+ANSI_COLOR_BRIGHT_BLUE=94
+ANSI_COLOR_BRIGHT_MAGENTA=95
+ANSI_COLOR_BRIGHT_CYAN=96
+ANSI_COLOR_BRIGHT_WHITE=97
+
+TRACE_LEVEL_QUIET=-1
+TRACE_LEVEL_STANDARD=0
+TRACE_LEVEL_VERBOSE=1
+TRACE_LEVEL_DEBUG=2
+TRACE_LEVEL=$TRACE_LEVEL_STANDARD
+
+function debug
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_DEBUG ]]
+	then
+		echo -e "[\e[${ANSI_COLOR_GREEN}mDEBUG  \e[00m] $@"
+	fi
+}
+
+
+function info
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_VERBOSE ]]
+	then
+		echo -e "[\e[${ANSI_COLOR_CYAN}mINFO   \e[00m] $@"
+	fi
+}
+
+function warning
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_STANDARD ]]
+	then
+		echo -e "[\e[${ANSI_COLOR_YELLOW}mWARNING\e[00m] $@" 1>&2
+	fi
+}
+
+function error
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_STANDARD ]]
+	then
+		echo -e "[\e[${ANSI_COLOR_BRIGHT_RED}mERROR  \e[00m] $@" 1>&2
+	fi
+}
+
+function fatal
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_STANDARD ]]
+	then
+		echo -e "[\e[${ANSI_COLOR_RED}mFATAL  \e[00m] $@" 1>&2
+	fi
+	exit 1
+}
+
+function check_cmds
+{
+	for cmd in $@
+	do
+		{ hash $cmd > "/dev/null" 2>&1 ; } || { fatal "$cmd command not installed" ; }
+	done
+}
+
+function usage
+{
+	cat <<- EOF
+		usage: daos-make.sh [OPTIONS] CMD
+
+		Build and install DAOS for running avocado functional tests
+
+		Options:
+		   -j, --jobs <job>         Allow N jobs at once; one job with no arg
+		   -f, --force              Overwrite the current install of DAOS
+		   -h, --help               show this help message and exit
+		   -V, --version            show version number
+		   -q, --quiet              quiet mode
+		   -v, --verbose            verbose mode
+		   -D, --debug              debug mode
+
+		Commands:
+		   install                  Build and install DAOS
+		   update                   Update current install of DAOS
+	EOF
+}
+
+function run
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_STANDARD ]]
+	then
+		"$@"
+	else
+		"$@" &> /dev/null
+	fi
+}
+
+OPTIONS=$(getopt -o "j:fhVvDq" --long "jobs:,force,help,version,verbose,debug,quiet" -- "$@") || exit 1
+eval set -- "$OPTIONS"
+while true
+do
+	case "$1" in
+		-j|--jobs) JOBS_NB="$2" ; shift 2 ;;
+		-f|--force) FORCE_INSTALL=true ; shift 1;;
+		-h|--help) usage ; exit 0;;
+		-V|--version) echo "daos-make.sh version=$VERSION" ; exit 0 ;;
+		-v|--verbose) TRACE_LEVEL=$TRACE_LEVEL_VERBOSE ; shift 1 ;;
+		-D|--debug) TRACE_LEVEL=$TRACE_LEVEL_DEBUG ; set -x ; shift 1 ;;
+		-q|--quiet) TRACE_LEVEL=$TRACE_LEVEL_QUIET ; shift 1 ;;
+		--) shift ; break ;;
+		*) fatal "unrecognized command line option" ;;
+	esac
+done
+
+[[ $1 ]] || fatal "Command not defined: install, update"
+CMD="$1"
+
+check_cmds $SCONS_EXE
+
+set -e
+
+info "Mount of home partition"
+{% if "daos_clients" in groups and groups["daos_clients"] | length > 0 %}
+run $CLUSH_EXE -l root -w $CLIENTS_LIST -w $SERVERS_LIST  mountpoint "$HOME"
+{% else %}
+run $CLUSH_EXE -w $SERVERS_LIST ldcondig mountpoint "$HOME"
+{% endif %}
```

Comment:
(lint) You need a space after the '{'. [SC1054]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]